.github/workflows/workflow.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Docker image
+
+on: [push, pull_request]
```

Comment:
search GitHub docs for info about dealing with "dangling images" - what happens when pull request get deleted etc. Probably you'll need to manually drop images from the registry

Reply:
I would like to see how it behaves after merge